yt/yt/server/query_tracker/profiler.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <yt/yt/library/profiling/sensor.h>
```

Comment:
yt/yt/client should be the first include, since yt/yt/library is more general than yt/yt/client